jenkins/promotion/promote-repos.jenkinsfile

```diff
@@ -28,18 +38,19 @@ pipeline {
         )
     }
     stages {
-        stage('promote yum repo') {
+        stage('promote repo') {
             agent {
                 docker {
                     label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
-                    image 'opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-build-v2'
+                    image docker_images["${DISTRIBUTION_REPO_TYPE}"]
```

Comment:
Where the type of these docker image get picked?

Reply:
Top of changes.
```

def docker_images = [
    "yum": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-build-v2",
    "apt": "opensearchstaging/ci-runner:ci-runner-ubuntu2004-opensearch-build-v1",
]
```